android/src/app/organicmaps/maps/DownloadResourcesLegacyActivity.java

```diff
@@ -1,4 +1,4 @@
-package com.mapswithme.maps;
+package app.organicmaps.maps;
```

Comment:
```suggestion
package app.organicmaps;
```
This is our current package name. Without .maps at the end.